corehq/apps/locations/static/locations/js/location.js

```diff
@@ -1,11 +1,17 @@
-/* globals TEMPLATE_STRINGS */
+/* globals django */
```

Comment:
fyi you can also deal with this by using `gettext` instead of `django.gettext`, because the former is in the allowed globals list in [.eslintrc.js](https://github.com/dimagi/commcare-hq/blob/master/.eslintrc.js#L17).

Why isn't `django` also in that list? No good reason. Maybe to nudge us towards using `gettext` alone, just for the sake of having a standard.